### ISSUE TITLE
 Hexerei 1.16.5 Crashes with Flash's NPCs 1.16.5

### ISSUE THREAD

**Issue by AuroraHydroid** [JoeFoxe/Hexerei-1.18#1]
When i load up a single player world, the game crashes and gives me this error:
"The game crashed whilst exception in server tick loop
Error: java.lang.NullPointerException: Exception in server tick loop
Exit Code: -1"

Quoted from someone who understands crash logs better than i do:
"a mod with the namespace herexi is colliding with flashnpc's core
it's saying here the crow is trying to construct its object, but that object becomes unusable to flash's npcs' enum creator because it's missing important data so flash's npc's is seeing a null 
long story short joefoxe didn't setup his entity's ai in a mod safe manner.
flash's npcs is expecting ai information to reside in the usual location that all LivingEntities should have it, so that it can build an enum list off that "

Attached is the crash log
[crash-2021-12-18_17.01.03-server.txt](https://github.com/JoeFoxe/Hexerei-1.18/files/7740040/crash-2021-12-18_17.01.03-server.txt)

**Comment by JoeFoxe** [comment]
honestly I think this just might be too much of an edge case for me to try to fix. I just dont have time to go back and fix a 1.16.5 edge case bug, sorry my man.
